manifest_staging/charts/secrets-store-csi-driver/templates/secrets-store-csi-driver.yaml

```diff
@@ -110,7 +110,10 @@ spec:
             - containerPort: {{ .Values.livenessProbe.port }}
               name: healthz
               protocol: TCP
-          livenessProbe:
+            - containerPort: {{ .Values.linux.metricsAddr }}
+              name: metrics
+              protocol: TCP
+           livenessProbe:
```

Comment:
The indentation is off here. This fails while trying `helm template`

Reply:
Doh, was to much in a rush when I did this sorry about that.
Thanks for pointing it out.